lib/tasks/import_cites_educatives.rake

```diff
@@ -21,13 +22,72 @@ task :import_cites_educatives => :environment do
         code_uai: row[4],
         coordinates: {latitude: coordinates[0], longitude: coordinates[1]}
       )
-      
+
       puts "OK #{school.id} - #{school.name}"
     else
       p "ERROR : " + address
     end
   end
 end
 
+# Temporary task
+desc 'Fix missing school street in production data'
+task :fix_missing_school_street => :environment do
```

Comment:
It doesn't work with a little bit of Geocoder for the missing streets ?

Reply:
could work, you're right : worth trying when sure by 100%, right ? @pierrax . But how to be sure by 100% ?